extensions/podman/src/extension.ts

```diff
@@ -76,6 +76,16 @@ const qemuHelper = new QemuHelper();
 const podmanBinaryHelper = new PodmanBinaryLocationHelper();
 const podmanInfoHelper = new PodmanInfoHelper();
 
+let shouldNotifySetup = true;
+const setupPodmanNotification: extensionApi.NotificationOptions = {
+  title: 'Podman needs to be set up!',
```

Comment:
```suggestion
  title: 'Podman needs to be set up',
```

Such a nitpick, but should we have exclamations / shouting in important notifications like this? WDYT @mairin